src/pages/extending-chat-widget/javascript-api/index.mdx

```diff
@@ -18,8 +18,6 @@ It might come in handy whenever you want to gather some additional data using Li
 
 This document focuses on Developers and requires a basic knowledge of JavaScript. However, if you would have any questions, don't hesitate to start a chat with our Support Team or add a new topic on [Developers Community](https://www.livechat.com/community/c/developers-api).
 
-This API works only with the [new Chat Widget](https://developers.livechat.com/blog/new-chat-window-look/).
```

Comment:
Connected with [this case](https://livechat.slack.com/archives/C2XMTMQ5S/p1663065102402759) on #platform-docs. Conformed with Konrad K. that all clients have a new one so we can remove this info:)